client/a8c-for-agencies/sections/agency-tier/overview-content/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Card, CardBody, __experimentalVStack as VStack } from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import InfluencedRevenue from './influenced-revenue';
+import TierCards from './tier-cards';
+import type { AgencyTier } from './types';
+
+export default function AgencyTierOverviewContent( {
+	currentAgencyTier,
+	totalInfluencedRevenue,
+}: {
+	currentAgencyTier?: AgencyTier;
+	totalInfluencedRevenue: number;
+} ) {
+	const isSmallViewport = useViewportMatch( 'huge', '<' );
```

Comment:
Nitpicking: Should we consider using isLargeViewport here to maintain consistency with the breakpoint analogy?